tools/cache.py

```diff
@@ -32,10 +32,7 @@ def try_remove_ending(thestring, ending):
       if not shared.Settings.WASM_BACKEND:
         dirname = os.path.join(dirname, 'asmjs')
       else:
-        if shared.Settings.EXPERIMENTAL_USE_LLD:
-          dirname = os.path.join(dirname, 'wasm-lld')
-        else:
-          dirname = os.path.join(dirname, 'wasm')
+        dirname = os.path.join(dirname, 'wasm')
```

Comment:
seems like this switches to use lld in a dir that used to be used for s2wasm? probably doesn't matter, but in theory if someone doesn't clear their cache perhaps there could be an issue? anyhow we can tag a new version soon, which would clear it automatically.

Reply:
Yes, that is correct.  I was wondering if there was a way to force the cache to be cleared.  Sounds like the version tag is the way to achieve that?

Reply:
Yeah, bumping the version is the mechanism, basically. It's a good time to tag one anyhow, I'll do it after this lands. However, there might be a temporary error on bots.